File: 1° Período - Introdução a Programação 1/Exercicios em Python/Lista 4 - Python/q9.py
#Crie um algoritmo que leia a temperatura de uma pessoa e mostre a mensagem “Febre Alta” (temp
#= 39), “Febril” (39 > temp = 37) ou “Sem Febre” (temp < 37).

temp=float(input("Insira a temperatura de uma pessoa: "))
if temp>=39:
    print("Febre alta")
elif temp>=37 and temp<39:
    print("Febril")
else:
    print("Não há febre")
    

File: 1° Período - Introdução a Programação 1/Exercicios em C/Lista 4 - C/q9.c
//Crie um algoritmo que leia a temperatura de uma pessoa e mostre a mensagem “Febre Alta” (temp
//= 39), “Febril” (39 > temp = 37) ou “Sem Febre” (temp < 37).

#include<stdio.h>
int main(){
	float temp;
	printf("Insira a temperatura da pessoa: ");
	scanf("%f", &temp);
	if(temp>=39){
		printf("Febre alta");
	}
	else if(temp>=37 && temp<39){
		printf("Febril");
	}
	else{
		printf("Não há febre");
	}
	    
}
